linera-execution/src/util/mod.rs

```diff
@@ -21,14 +21,6 @@ pub trait UnboundedSenderExt<Request> {
     ) -> Result<oneshot::Receiver<Response>, ExecutionError>
     where
         Response: Send;
-
```

Comment:
This cleanup belongs to a separate PR for proper review/discussion in light of #1416

Reply:
I'd like to factor it into a separate commit so we can choose to revert it later, but it's going to get squashed away anyway 😅

I guess I can comment it out or `#[allow(dead_code)]` for now to minimize the changeset to be manually reverted later.

Reply:
I added a `TODO` and an `allow(dead_code)` for now.